packages/components/pages/f-self-exclusion/src/components/SelfExclusion.vue

```diff
@@ -253,6 +256,10 @@ export default {
             return this.exclusions.find(exclusion => exclusion.type === 'alcohol');
         },
 
+        exclusionPeriodExpired () {
```

Comment:
My Vue is a little rusty but shouldn't these two be moved to a getter and then you read there? Something like below. You can then read direct from the getter within this file? I think you can also move the method above into that too and reference the places it needs to be used via the getter instead.

```javascript
        exclusionPeriodExpired: (state, getters) => getters.alcoholExclusion.state === 'exclusionPeriodExpired',

```

Reply:
Is there any purpose for this change? It is implemented to follow up the pattern for the rest of the component.

Reply:
Happy for you to look in a different PR but i think it's a cleaner approach.